app/components/FacebookMessengerLogo.js

```diff
@@ -0,0 +1,28 @@
+const React = require('react');
+const classnames = require('classnames');
+
+module.exports = React.createClass({
+    displayName: 'FacebookMessengerLogo',
+
+    render() {
+        const classes = classnames(
+            'tilt-icon',
+            'tilt-icon-facebook-messenger-logo',
+            {
+                [this.props.className]: !!this.props.className
+            }
+        );
```

Comment:
`const classes = classnames('tilt-icon', 'tilt-icon-facebook-messenger-logo', this.props.className);`